### PR TITLE
UML-1480: Update page when LPA can not be found on requesting an activation key

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose
https://opgtransform.atlassian.net/browse/UML-1480

Fixes UML-1480

## Approach

Just content update with additional data passed into twig and translations

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [x] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [x] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig [hunk 1]
```diff
@@ -35,25 +35,69 @@
...
+
+                        <details class="govuk-details" data-module="govuk-details">
+                            <summary class="govuk-details__summary" role="button">
+                        <span class="govuk-details__summary-text">
+                          {% trans %}What to do if you’re sure you entered the right information{% endtrans %}
+                        </span>
+                            </summary>
+                            <div class="govuk-details__text">
+                                <p>{% trans %}If you're sure you entered the right information, please call us:{% endtrans %}</p>
+                                <p>{% trans %}Telephone: 0300 456 0300{% endtrans %}</p>
```
Comment (lukejolliffe) [inline service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig:97]: These should use the telephone contact details template so all the information can still be updated all together

**Review by lukejolliffe — APPROVED** [review]

File: service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig (hunk 1 above)
Comment (lukejolliffe) [inline service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig:97]: These should use the telephone contact details template so all the information can still be updated all together

**Comment by SeemaMenon** [comment]
Screenshot 2022-01-19 at 15 53 41
Screenshot 2022-01-19 at 15 53 49
Screenshot 2022-01-19 at 15 54 03
Screenshot 2022-01-19 at 15 54 13

File: service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig [hunk 2]
```diff
@@ -32,25 +32,68 @@
...
+
+                        <details class="govuk-details" data-module="govuk-details">
+                            <summary class="govuk-details__summary" role="button">
+                        <span class="govuk-details__summary-text">
+                          {% trans %}What to do if you’re sure you entered the right information{% endtrans %}
+                        </span>
+                            </summary>
+                            <div class="govuk-details__text">
+                                <p class="govuk-body">{% trans %}If you're sure you entered the right information, please call us:{% endtrans %}</p>
+                                {{ include('@partials/contact-details/telephone.html.twig') }}
```
Comment (lukejolliffe) [inline service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig:94]: The title false attribute is needed to get the telephone and textphone to appear without a heading

```suggestion
                                {{ include('@partials/contact-details/telephone.html.twig', {title: false) }}
```

**Review by lukejolliffe — CHANGES_REQUESTED** [review]

File: service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig (hunk 2 above)
Comment (lukejolliffe) [reply service-front/app/src/Actor/templates/actor/cannot-find-lpa.html.twig:94]: Actually looking at the screenshots it should be fine without?

**Review by lukejolliffe — APPROVED** [review]

**Pull request merged**